.github/workflows/pr-check.yml

```
@@ -0,0 +1,46 @@
+  pull_request:
+    branches:
+      - preview  # Trigger only on pull requests targeting the preview branch
+
+jobs:
+  build:
+    name: Build and Up-to-Date Check
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)